### DISCUSSION TITLE
How to disable adding 'prefix' when generate SQL?

### DISCUSSION THREAD

**Discussion by opensplitter** [dbeaver/dbeaver#18143] [Q&A]
Hello. 
I am using DBeaver version 21.1.1 with Sybase driver.
When I generate SQL by right-clicking data row>Generate SQL> or Advanced Copy as SQL, the output as,
`
INSERT INTO sys_msg
(app_id, f_id, op_id, s_code, m_code, description, cause, ..)
VALUES(1, 4400, 210, N'A', N'029', N'DESCRIPTION', N'Fail to', ...);
`

Is there any setting to avoid adding 'N' prefix, so that the generated SQL is simply like 
`
INSERT INTO sys_msg
(app_id, f_id, op_id, s_code, m_code, description, cause, ..)
VALUES(1, 4400, 210, 'A', '029', 'DESCRIPTION', 'Fail to', ...);
`

I have tried to change 'Edit Driver Settings' > 'Advanced paramters' but seems no option for unicode prefix.
Would anyone can help? 
thanks.

**Comment by LonwoLonwo** [comment]
Hello @opensplitter 

What is the reason for removing the N prefix?

**Comment by opensplitter** [comment]
Hi LonwoLonwo.
As my working partners’ workstations can generate SQL without the N prefix and we do not need this for the cicd. Thus, I doubt if it is an option to turn it off then I do not need to remove them manually…

**Comment by LonwoLonwo** [comment]
Ok. You can create a feature request ticket on this option for SQL Server here: https://github.com/dbeaver/dbeaver/issues

**Comment by LonwoLonwo** [comment]
And yes, for now, DBeaver does not have such an option.
